### PR TITLE
Add Scala to the list of languages

### PR DESCRIPTION
**Pull request by azolotko**

### CODE REVIEW

**Comment by codicocodes** [comment]
Thank you for the contribution! I will do some testing

File: packages/back-nest/src/challenges/services/ts-parser.factory.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import * as rb from 'tree-sitter-ruby';
 import * as cs from 'tree-sitter-c-sharp';
 import * as go from 'tree-sitter-go';
 import * as rs from 'tree-sitter-rust';
+import * as scala from 'tree-sitter-scala';
```
Comment (codicocodes) [inline packages/back-nest/src/challenges/services/ts-parser.factory.ts:15]: Missing `tree-sitter-scala` in `package.json`

**Comment by codicocodes** [comment]
Thanks for the contribution!

File: packages/back-nest/src/challenges/services/ts-parser.factory.ts (hunk 1 above)
Comment (azolotko) [reply packages/back-nest/src/challenges/services/ts-parser.factory.ts:15]: Sorry, missed it while cherry-picking changes 😞

**Pull request merged**